content/en/service_catalog/investigating.md

```diff
@@ -27,24 +27,30 @@ further_reading:
 algolia:
   tags: ['service catalog']
 ---
+## Assigning an owner to a service:
+You can assign a ‘team’ to entries in the Service Catalog either in the UI or by creating a [Service Definition][4]. It’s recommended that you set up [Datadog Teams][5] so you may specify individual members of the team and take advantage of *Teams* filters across common views like Dashboards and Notebook lists. 
```

Comment:
```suggestion
You can assign a `team` to entries in the Service Catalog either in the UI or by creating a [Service Definition][4]. Datadog recommends that you set up [Datadog Teams][5] so that you can specify individual members of the team and take advantage of *Teams* filters across common views like Dashboards and Notebook lists. 
```